### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by joffinjoy**

### CODE REVIEW

File: Dockerfile-README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Specifications
+
+1.  Location: **_Root directory of the service._**
+2.  File name: **_Dockerfile_**
+
+<br>
+
+## Expected Root Directory Structure
+
+    ./notification/
```
Comment (rakeshSgr) [inline Dockerfile-README.md:14]: @joffinjoy it should be /documents/

File: Service-Standardization-README.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+                    }
+                }
+            ]
+        }
+        ```
+
+    -   Add **.eslintrc.json** file to src directory of service:
+
+        ```
+        ./notification/src
```
Comment (rakeshSgr) [inline Service-Standardization-README.md:77]: @joffinjoy Change to documents

File: Dockerfile-README.md (hunk 1 above)
Comment (joffinjoy) [reply Dockerfile-README.md]: @rakeshSgr  Assuming document service wouldn't be an apt representation of usual Elevate services, setting the service name to more generic <service-name>.

File: Service-Standardization-README.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+1.  Install Prettier & Eslint:
+
+    ```
+    ELEVATE/<service-name>/src$ npm install prettier eslint --save-dev
+    ```
+
+    -   Add **.prettierrc.json** file to root of service directory:
+
+        ```
+        ./notification/
```
[the remark this answers is not in the document]
Comment (joffinjoy) [reply Service-Standardization-README.md]: @rakeshSgr  Assuming document service wouldn't be an apt representation of usual Elevate services, setting the service name to more generic <service-name>.

File: Service-Standardization-README.md (hunk 2 above)
Comment (joffinjoy) [reply Service-Standardization-README.md]: @rakeshSgr  Assuming document service wouldn't be an apt representation of usual Elevate services, setting the service name to more generic <service-name>.

File: Docker-compose-README.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+
+**Note:** It isn't always necessary to run **down** command. Existing containers and networks can be stopped gracefully by using **Ctrl + C** key combination.
+
+**Warning:** Do not use docker-compose in production.
+
+<br>
+
+## Expected Micro-services Directory Structure
+
+Every micro-service directory should be placed in the same root directory as that of **mentoring service** and mentoring service's root directory holds the **docker-compose.yml** file.
```
[the remark this answers is not in the document]
Comment (joffinjoy) [reply Docker-compose-README.md:30]: Didn't understand this one. This is the documentation for Docker-compose that exists in mentoring's root directory.

File: Docker-compose-README.md [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+This directory structure ensures proper relative paths from docker-compose file to appropriate micro-services.
+
+Each micro-service which should be part of docker-compose must have a Dockerfile in its root directory.
```
[the remark this answers is not in the document]
Comment (joffinjoy) [reply Docker-compose-README.md:34]: @rakeshSgr Didn't understand this one. This is the documentation for Docker-compose that exists in mentoring's root directory. Nothing to do with document service. Can you please clarify?

**Pull request closed without merging**